Framework/PythonInterface/mantid/kernel/src/Exports/V3D.cpp

```diff
@@ -63,10 +63,13 @@ void export_V3D() {
       .def(self + self)
       .def(self += self)
       .def(self - self)
+      // cppcheck-suppress duplicateExpression
```

Comment:
These are better addressed by following the example @AndreiSavici gave in issue #15281.